stream/models.py

```diff
@@ -46,6 +46,86 @@ def __str__(self):
             self.reporter.username, self.content_type, self.content_id)
 
 
+NOTIFICATION_TEMPLATES = (
+    # Profile
+    (0, '{} has subscribed to you'),
```

Comment:
This shouldve been (0, '{obj.owner.username} has subscribed to you').

The property would then be NOTIFICATION_TEMPLATES[0][1].format(obj=self.target_object)